client/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components/macro'
+import React from 'react'
+
+function Footer({ onCreateQuestion }) {
+  return (
+    <Wrapper onSubmit={e => handleSubmit(e)}>
+      <Label aria-label="Add new question">
+        <TextInput
+          name="question"
+          placeholder="Enter your question here!"
+          type="text"
+        />
+      </Label>
+      <SubmitButton aria-label="Submit question">Submit</SubmitButton>
```

Comment:
The `aria-label` here is not super necessary as `Submit` should be sufficient enough for users to understand. Now screen reader user will get `Submit question` and everyone else `Submit`.